python-package/xgboost/spark/core.py

```diff
@@ -1092,12 +1093,87 @@ def _get_feature_col(
             )
         return features_col, feature_col_names
 
+    def _get_pred_contrib_col_name(self) -> str:
+        """Return the pred_contrib_col col name"""
+        pred_contrib_col_name = ""
```

Comment:
Let's make this return `Optional[str]` instead of `str.  A `None` is easier to recognize than an empty string.

Reply:
Good suggestion. Done